optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/employee/EmployeeRestService.java

```diff
@@ -55,27 +53,27 @@ Employee getEmployee(@ApiParam(required = true) @PathParam("tenantId") Integer t
     @POST
     @Path("/add")
     Employee addEmployee(@ApiParam(required = true) @PathParam("tenantId") Integer tenantId,
-            @ApiParam(value = "with no id", required = true) Employee employee);
-    
+                         @ApiParam(value = "with no id", required = true) Employee employee);
+
     /**
      * @param employee never null
      * @return never null, with an updated {@link AbstractPersistable#getVersion()}
      */
     @ApiOperation("Update an employee")
     @POST
```

Comment:
Isn't it more idiomatic to make this a PUT request against the root employee path (/tenant/{id}/employee/)?